src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -33,4 +38,16 @@ public void updateName(String name) {
     public void updateColor(String color) {
         this.color = color;
     }
+
+    public void validateLastStation() {
+        if (this.getSections().getDownStationIds().size() < 1) {
```

Comment:
size 1이하이면 값이 없는 상태같은데요. 
아래와 같이 코드가 작성될수있을 것 같아요 ~ 
```suggestion
       if (this.getSections().getDownStationIds().isEmpty() 
```